src/features/campaigns/components/CampaignDetailsForm.tsx

```diff
@@ -31,25 +30,20 @@ const CampaignDetailsForm = ({
       : null
   );
 
-  const initialValues = {
-    info_text: campaign?.info_text,
-    status: campaign?.published ? 'published' : 'draft',
-    title: campaign?.title,
-    visibility: campaign?.visibility,
-  };
+  const [title, setTitle] = useState<string>(campaign?.title ?? '');
+  const [infoText, setInfoText] = useState<string>(campaign?.info_text ?? '');
+  const [status, setStatus] = useState<'published' | 'draft'>(
+    campaign?.published ? 'published' : 'draft'
+  );
+  const [visibility, setVisibility] = useState<'hidden' | 'open'>(
+    (campaign?.visibility as 'hidden' | 'open') ?? 'hidden'
```

Comment:
This makes me think that the `visibility` property on the `ZetkinCampaign` type is too loosely typed. Not the problem of this PR but I'm noting it down for something to look into!